packages/amplify-e2e-core/src/configure/index.ts

```diff
@@ -66,6 +66,7 @@ export function amplifyConfigure(settings: AmplifyConfiguration): Promise<void>
       .wait('Successfully set up the new user.')
       .run((err: Error) => {
         if (!err) {
+          injectSessionToken();
```

Comment:
The callback passed to `run` gets called after the command is executed. So I think the injectSessionToken should probably be called before `run`

Reply:
as long as the access/secret keys get set and the file gets created before we call `run` that's fine with me. But it does work when implemented this way.

Reply:
If you're using the standard ENV variable name I think the SDK probably picks it up

Reply:
That probably is why it works this way. I think the amplify-cli codebase though reads from the config file, because these tests do not pass when I don't include this injection.